katalog/calico/sm.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    k8s-app: calico-node
+  name: calico-node
+  namespace: monitoring
```

Comment:
No need to put the ServiceMonitor in the `monitoring` namespace, furthermore this gets overwritten by kustomize.